README.md

```diff
@@ -0,0 +1,32 @@
+# Open Liberty Javadoc 
+This repository contains the Javadoc that is published on https://openliberty.io, including the following models:
+
+- Open Liberty APIs
+- Open Liberty SPIs
+- MicroProfile APIs
+- Jakarta EE APIs
+- Java EE APIs
+
+Microprofile and Jakarta API documentation is updated with each major release from those platforms. The Open Liberty API and SPI documentation is updated with each 4-week release of the runtime. 
+
+## Preparing Open Liberty API and SPI Javadoc files for publication
+
+The following steps document the procedure the Open Liberty Docs team follows to prepare the Open Liberty API and SPI Javadoc files for publication with each release. 
+
+Complete these steps 2-3 days before the release to ensure that you have time to verify the Javadoc on the staging site. However, the Javadoc doesn't appear on the staging site until after the other autogenerated files for the release are committed to the [staging branch of the docs-generated](https://github.com/OpenLiberty/docs-generated/tree/staging) repository and the resulting [staging site](https://docs-staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/) build finishes. The autogenerated files are typically added to the `staging` branch of the `docs-generated` repo 1-2 days before the release.
+
+1. Clone the `docs-javadoc` repository 
+2. Get the generated `com.ibm.openliberty.javadoc.doc.jar` file from the Liberty GM build.
+3. Change the extension of the file from `com.ibm.openliberty.javadoc.doc.jar` to `com.ibm.openliberty.javadoc.doc.zip`.
+4. Extract the `com.ibm.openliberty.javadoc.doc.zip` to a temporary directory on your local machine.
+5. Create a branch off the `prod` branch of `docs-javadoc`.
+6. In the `modules/reference` directory, create a new folder and name it based on the version for that release, for example `modules/reference/23.0.0.7`.
+7. Copy all the `-javadoc` folders from your local extracted copy of the Javadoc file into the new `modules/reference/<version>` folder that you created in the previous step.
+8. Commit your changes to the branch that you created in step 5.
+9. Create a pull request from your branch to `staging`. Do not add the files to `draft`.
+10. Get the PR reviewed by a docs team member and merge it.
```

Comment:
@cbridgha when 'you' push the docs-generated content, are doing that manually, or is it ran as part of a piece of automation?
It feels like the upload of both the docs-generate and the Liberty API/SPI (steps 1-10 above) could be automated.  From my understanding, the GM automation build already generates some PRs, so I would hope that most of the effort to automate these two tasks is largely already in place?